atomix/pom.xml

```diff
@@ -0,0 +1,283 @@
+        <scope>test</scope>
+        <version>${concurrentunit.version}</version>
+      </dependency>
+      <dependency>
+        <artifactId>mockito-core</artifactId>
+        <groupId>org.mockito</groupId>
+        <scope>test</scope>
+        <version>${version.mockito}</version>
+      </dependency>
+      <dependency>
+        <artifactId>guava</artifactId>
+        <groupId>com.google.guava</groupId>
+        <version>${version.guava}</version>
+      </dependency>
+      <dependency>
+        <artifactId>commons-lang3</artifactId>
+        <groupId>org.apache.commons</groupId>
+        <version>${commons.lang3.version}</version>
+      </dependency>
+      <dependency>
+        <artifactId>commons-math3</artifactId>
+        <groupId>org.apache.commons</groupId>
+        <version>${commons.math3.version}</version>
+      </dependency>
+      <dependency>
+        <artifactId>kryo</artifactId>
+        <groupId>com.esotericsoftware</groupId>
+        <version>${kryo.version}</version>
+      </dependency>
+      <dependency>
+        <artifactId>config</artifactId>
+        <groupId>com.typesafe</groupId>
+        <version>${config.version}</version>
+      </dependency>
+      <dependency>
+        <artifactId>classgraph</artifactId>
+        <groupId>io.github.classgraph</groupId>
+        <version>${classgraph.version}</version>
+      </dependency>
+
+
+      <dependency>
+        <artifactId>error_prone_annotations</artifactId>
+        <groupId>com.google.errorprone</groupId>
+        <version>${version.errorprone}</version>
+      </dependency>
+
+      <dependency>
+        <artifactId>objenesis</artifactId>
+        <groupId>org.objenesis</groupId>
+        <version>${version.objenesis}</version>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
+  <build>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <artifactId>buildnumber-maven-plugin</artifactId>
+          <configuration>
+            <doCheck>false</doCheck>
+            <doUpdate>false</doUpdate>
+            <format>{0} (revision {1} built on {2,date,yyyy-MM-dd HH:mm:ss})</format>
+            <items>
+              <item>${project.version}</item>
+              <item>scmVersion</item>
+              <item>timestamp</item>
+            </items>
+            <revisionOnScmFailure>N/A</revisionOnScmFailure>
+            <shortRevisionLength>6</shortRevisionLength>
+          </configuration>
+          <executions>
+            <execution>
+              <goals>
+                <goal>create</goal>
+              </goals>
+              <id>configure-buildnumber</id>
+              <phase>validate</phase>
+            </execution>
+          </executions>
+          <groupId>org.codehaus.mojo</groupId>
+          <version>1.4</version>
+        </plugin>
+      </plugins>
+    </pluginManagement>
+    <plugins>
+      <!-- LICENSE PLUGIN -->
+      <!-- EXCLUDE EVERYTHING FOR NOW -->
+      <plugin>
```

Comment:
So what does the license plugin do for now? Nothing? How much work would it be to have at least the check?

Reply:
This assumption is not correct also the comment above is out dated sorry for that. 
I verified it with an extra branch where I added a empty class https://github.com/zeebe-io/zeebe/commit/cd34694ae19fec1299cbe8524a6d6bcb45597e06

The build check fails as expected https://ci.zeebe.camunda.cloud/job/zeebe-io/job/zeebe/job/zell-license-check/1/console
```

[INFO] 
[INFO] --- maven-surefire-plugin:3.0.0-M4:test (default-test) @ zeebe-protocol-asserts ---
[WARNING] Missing header in: /home/jenkins/agent/workspace/eebe-io_zeebe_zell-license-check/atomix/storage/src/main/java/io/atomix/storage/journal/TestLicense.java
[INFO] Tests are skipped.
[INFO] 
```

If we run maven locally it will add the missing header:


```diff
diff --git a/atomix/storage/src/main/java/io/atomix/storage/journal/TestLicense.java b/atomix/storage/src/main/java/io/atomix/storage/journal/TestLicense.java
index 09ddcf371..addecb07b 100644
--- a/atomix/storage/src/main/java/io/atomix/storage/journal/TestLicense.java
+++ b/atomix/storage/src/main/java/io/atomix/storage/journal/TestLicense.java
@@ -1,5 +1,18 @@
+/*
+ * Copyright © 2020 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
 package io.atomix.storage.journal;
 
-public class TestLicense {
-
-}
+public class TestLicense {}
```

Reply:
Ah, yeah, I assumed because of the comment, sorry

Reply:
It's good for now, I still wish we had some way to also check if the wrong license header is there, but we can do that after.

Reply:
I think if you add a wrong header it will also fail the build